dune_client/client.py

```diff
@@ -132,8 +146,13 @@ def cancel_execution(self, job_id: str) -> bool:
         except KeyError as err:
             raise DuneError(response_json, "CancellationResponse", err) from err
 
-    def _refresh(self, query: Query, ping_frequency: int = 5) -> str:
-        job_id = self.execute(query).execution_id
+    def _refresh(
+        self,
+        query: Query,
+        ping_frequency: int = 5,
+        performance: Union[str, None] = None,
```

Comment:
Nit: Is there any reason you prefer `Union` over `Optional[str]` or even `str | None`? I guess we are still supporting python 3.7 so `str | None` wont work.

Reply:
No reason tbh, will change to `Optional[str]`

Reply:
Changed